bloom-housing/bloom
# fix: include demographics in partners applicant csv export

# Pull Request Template

## Issue Overview

This PR addresses [#493](https://github.com/metrotranscom/doorway/issues/493)


## Description

Adds new flag `enablePartnerDemographics`. When `true` on csv export it will return also demographics for partners.

## How Can This Be Tested/Reviewed?

in partners log in as partner (`partner-user@example.com`). Without changes in db it should not return demographics in csv applications export (demographics are at the end of csv file). Now change flag `enablePartnerDemographics` for jurisdiction of listing to `true`. Now demographics should be returned for `partners`.
For admins demographics should be returned irrespectively of flag value.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
